ui/src/components/denali/icons/Icon.js

```diff
@@ -59,12 +59,15 @@ Icon.propTypes = {
     onClick: PropTypes.func,
     isLink: PropTypes.bool,
     innerRef: P.oneOfType([P.func, P.object]),
+    /*By default icon will display name on hover. Setting this to false will stop it*/
+    enableTitle: PropTypes.bool
 };
 
 Icon.defaultProps = {
     icon: 'x',
     size: '1em',
     verticalAlign: 'text-bottom',
+    enableTitle: true
```

Comment:
I have set the default behaviour to true to prevent changes in other users of Icon.js. However, I can see that there are locations that might benefit from setting it to false (such as the icons that appear in the list of role members)